STAR.Web/Views/Shared/ContractorListPartial.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@model IEnumerable<Contractor>
```

Comment:
Move this code to the view itself, and remove this partial.  We may have pre-optimized.